packages/migration/src/mods/7.0.0/renameReactDeprecated.ts

```diff
@@ -0,0 +1,25 @@
+import { mod } from 'riceburn';
+import ts from 'typescript';
+import { migration } from '../../migration';
+
+export default migration('rename componentWillReceiveProps with UNSAFE_componentWillReceiveProps', () => {
+  mod('**/*.ts?(x)').asTypescript((node, modder) => {
```

Comment:
Why the `?` after `s`?

Reply:
Minimatch syntax is different than regex apparently... I had to look it up 😅